ledfx/sendspin/stream.py

```
@@ -463,6 +477,59 @@ def _stream_clear_handler(self, roles):
+        if self._client is None or self._loop is None:
+            return
+        try:
+            asyncio.run_coroutine_threadsafe(
+                self._client.send_player_state(
+                    state=PlayerStateType.SYNCHRONIZED,
+                    volume=self._ma_volume,
+                    muted=self._ma_muted,
+                ),
+                self._loop,
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)